server/config/auth/passportConfig.js

```diff
@@ -1,10 +1,20 @@
 import passport from 'passport';
 import passportJWT from 'passport-jwt';
 import dotenv from 'dotenv';
-import users from '../../db/controllers/users';
-
+import User from '../../db/controllers/users';
+import knexModule from 'knex';
+import bookshelfModule from 'bookshelf';
+import { development as devconfig } from '../../../knexfile';
+import bookshelfBcrypt from 'bookshelf-bcrypt';
+const { bookshelf } = require('../../db/init').default;
```

Comment:
Why don't we use the named export convention in init, then we can just do
`import { bookshelf } from '../../db/init'`